atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -102,6 +103,7 @@
     Map<InetSocketAddress, CassandraClientPoolingContainer> currentPools = Maps.newConcurrentMap();
     final CassandraKeyValueServiceConfig config;
     final ScheduledThreadPoolExecutor refreshDaemon;
+    private final Optional<MetricRegistry> metricRegistry;
```

Comment:
Would prefer we don't use optional for field, and in fact would like to require metrics